widget/embedded/src/Widget.tsx

```diff
@@ -21,6 +21,9 @@ const MainContainer = styled('div', {
   display: 'flex',
   justifyContent: 'center',
   alignItems: 'center',
+  '*': {
```

Comment:
It works but it affects all the elements (even if for those that don't need) so it causes some performance issues.

Couldn't we apply only to `html` and `body`? 
Or if we want to reset elements and only set to inherit fonts to read from `html` or `body` ?
There are some solutions like https://meyerweb.com/eric/tools/css/reset/.

Reply:
In the past, we used to change the body of the whole page with add/remove class. I think it's not appropriate to change the body. I remove * and allow elements to inherit from their parents.